README.md

```diff
@@ -29,6 +29,9 @@ import jubilant
 juju = jubilant.Juju()
 juju.deploy('snappass-test')
 juju.wait(jubilant.all_active)
+
+# Or only wait for specific applications:
+juju.wait(lambda status: jubilant.all_active(status, 'snappass-test'))
```

Comment:
Maybe something like this to be more explicit?

```suggestion
juju.wait(lambda status: jubilant.all_active(status, 'snappass-test', 'another-app'))
```

Reply:
Good call, done